### DISCUSSION TITLE
Closing window takes a while and shows warning - glium has triggered an OpenGL error during initialization.

### DISCUSSION THREAD

**Discussion by santokalayil** [emilk/egui#793] [Q&A]
While trying to close the gui window, It takes a while to close.. also in the command prompt it shows the following:
'''glium has triggered an OpenGL error during initialization. Please report this error: https://github.com/glium/glium/issues'''

I am using Windows 10

**Comment by emilk** [comment]
Hopefully this will be solved with https://github.com/emilk/egui/pull/685

**Reply by santokalayil** [reply]
Thank you very much for responding. But please know that I am not that expert. Did you mean that the issue is fixed now.?. Or what? I tried reading the full issue you have given.. since I am new to rust and gui programming, I did not find any solution.. I hope that the new changes requested by you to master branch will fix the issue..
